bench/dune

```diff
@@ -0,0 +1,20 @@
+(executable
+ (name bench)
+ (public_name bench)
+ (package ocamlformat-bench)
+ (libraries bechamel bechamel-js ocamlformat_lib stdio))
+
+(alias
+ (name bench)
+ (deps ./bench.exe))
+
+(rule
+ (alias runbench)
+ (action
+  (run ./bench.exe)))
+
+(rule
+ (alias runtest)
+ (package ocamlformat-bench)
+ (deps bench.exe)
+ (action (progn)))
```

Comment:
This rule is testing that the benchmark code compiles ?
Is it useful if the benchmarks are run in CI anyway ?

Reply:
I think that's actually a very good idea :)

Reply:
I admit I just picked it from `mirage/index` that is also using current-bench